.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,25 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches:
+    - "master"
+    - "*-draft"
```

Comment:
These branches are already push protected.
Why have code checks on push for the only branches that can not be directly pushed to?
Consider checking the project after pushes to all branches.

Reply:
I guess there is no reason to check pushes at all. I disabled action on pushes, because we only care about passing all checks on push-protected branches anyway, and these are already covered by `pull_request` trigger.